lib/signaling/v2/peerconnection.js

```diff
@@ -606,6 +612,10 @@ class PeerConnectionV2 extends StateMachine {
    * @returns {void}
    */
   _handleSignalingStateChange() {
+    if (this._peerConnection.signalingState === 'stable') {
+      this._appliedTrackIdsToAttributes = new Map(this._trackIdsToAttributes);
```

Comment:
just curious - why update  this._appliedTrackIdsToAttributes here, instead of saving them before calling `_setSimulcast` in offer and answer?

Reply:
@makarandp0 We only update the map after both offer and answer are applied. If we have to roll back the offer, then we would just discard the changes in `_trackIdsToAttributes`.